cmake/ctest/drivers/geminga/cron_driver.sh

```diff
@@ -39,20 +39,26 @@ export TDD_HTTPS_PROXY=$https_proxy
 SCRIPT_DIR=`cd "\`dirname \"$0\"\`";pwd`
 
 
-export export MODULEPATH=$SCRIPT_DIR:$MODULEPATH
+# Trilinos source repo
+export TRILINOS_SOURCE=$SCRIPT_DIR/../../../..
+
+# folder with the machine specific build info
+export BUILDS_DIR=$TRILINOS_SOURCE/cmake/ctest/drivers/$HOSTNAME
+
+# So we can load the env module specific to the build
+export MODULEPATH=$BUILDS_DIR:$MODULEPATH
+echo "Loading modules from $MODULEPATH"
+
+# update the Trilinos source
+pushd $TRILINOS_SOURCE
+./cmake/tribits/python_utils/gitdist --dist-no-color fetch
+popd
 
-# If you update the list of modules, go to ~/code/trilinos-test/trilinos/ and
-# do "git pull". Otherwise, the tests could fail on the first night, as we
-# would first run old cron_driver.sh and only then pull
 
 # ===========================================================================
 export CTEST_CONFIGURATION="default"
-# module load sems-cmake/3.17.1
-# module load sems-gcc/5.3.0
-# module load sems-openmpi/1.10.1
-# module load sems-superlu/4.3/base
-# module load sems-git/2.10.1
 module load muelu-gcc
```

Comment:
Does this 'muelu=gcc' module use the new SEMS modules?

Reply:
Yes, I added a module file to the same folder in an earlier PR.